src/javascripts/app.js

```diff
@@ -0,0 +1,17 @@
+var React = require('react');
+
+
+// Here is two empty lines, baddddd
+class Test {
```

Comment:
'class' is only available in JavaScript 1.7.<br>'Test' is defined but never used.